src/json2csv.ts

```diff
@@ -457,7 +457,7 @@ export const Json2Csv = function(options: FullJson2CsvOptions) {
      */
     function convert(data: object[]) {
         // Single document, not an array
-        if (utils.isObject(data) && !data.length) {
```

Comment:
`data.length` of an empty array is `0` and `!0` is `true` which is the source of the bug.